lm_eval/tasks/mednli.py

```diff
@@ -0,0 +1,41 @@
+"""
+
+"""
+import numpy as np
+from lm_eval.base import rf, BioTask
```

Comment:
`BioTask` was removed from the core task API to reduce bloat. I recall this being a special sub-class of PromptSourceTask that handled custom `promptsource` template discovery. That lookup has been removed as well, but we can probably expose [`_get_templates_from_task`](https://github.com/bigscience-workshop/lm-evaluation-harness/blob/1c1667a013526c1c7a90037bb8d109bb6fdab832/lm_eval/tasks/__init__.py#L299) as a way of handling that behavior. If you could point me to where the corresponding prompts are, I'd be glad to hook things up.

Reply:
They’re [here](https://github.com/OpenBioLink/promptsource/tree/main/promptsource/templates)

Reply:
Can you double-check that the linked branch contains the prompt templates intended for use with these tasks? 
- It's missing `mednli` templates.
- The `gad` task implementation has mismatching template paths ("gad_blurb_bigbio_text" is not in the linked branch but "gad_fold{N}_bigbio_text" is).
- The repo is on the incorrect branch; the harness is only compatible with `eval-hackathon` branch of `promptsource` because it contains multi-reference target support.

Reply:
Hmmm the [eval hackathon](https://github.com/OpenBioLink/promptsource/tree/eval-hackathon/promptsource/templates) branch also seems to have problems.

@hakunanatasha can you comment?

Reply:
@StellaAthena from what I gather, this is the branch where the templates were generated. Back when we were using the bigbio branch of lm-eval, this is the promptsource fork we referenced.

In theory, if we migrated appropriate template files over - is there a reason why we need this dependency? For example, if we hooked up to the default promptsource and regenerated all these files as custom .ini, can we not run lm-eval out of the box then specifically for BLOOM? I could be misunderstanding something. 

@jason-fries might have more insight

Reply:
To add also - we may not have evaluated all of these prompts, hence some act as "placeholders"; I did not run the eval component so I am not sure.